bin/stashcp

```diff
@@ -141,6 +143,19 @@ def doStashCpSingle(sourceFile, destination, cache, debug=False):
         es_send(payload)
 
 
+def parse_job_ad():
+    temp_list = {}
+    try:
+        with open(os.environ['_CONDOR_JOB_AD']) as job_file:
+            for line in job_file.readLines():
+                match = re.match('Owner|ProjectName', line, re.IGNORECASE)
+                if match:
+                    temp_list[match.group()] = line
```

Comment:
I think this is the right approach.

However, I'm not sure if this quite works?  Did you test this?  Seems like it's going to put the full line in as the group.  It also still keeps quotes.

You probably want a regex like this:
```
r = re.compile('^\s*(Owner|ProjectName)\s=\s"(.*)"', re.IGNORECASE)
```

Seems that we need a small unit test for this function, given it's not 100% clear what the output is.